cameradar_standalone/src/cachemanager.cpp

```diff
@@ -105,13 +102,25 @@ cache_manager_iface* cache_manager::operator->() { return this->ptr; }
 
 const cache_manager_iface* cache_manager::operator->() const { return this->ptr; }
 
-bool operator==(std::nullptr_t nullp, const cache_manager& p) { return p.ptr == nullp; }
```

Comment:
Should such a source code layout change for equality comparison functions belong to an other commit?